settings.gradle.kts

```diff
@@ -8,8 +8,8 @@ include("runners:cli")
 include("runners:maven-plugin")
 include("plugins:xml")
 include("plugins:mathjax")
-include("integration-tests:gradle-integration-tests")
-
+include("plugins:kotlin-as-java")
+//include("integration-tests:gradle-integration-tests")
```

Comment:
Don't comment it, if you want to skip tests better to uncomment a line in integration-tests' build.gradle